### PR TITLE
[PHPUnitBridge] Fix max threshold when only ignoreFile is set in SYMFONY_DEPRECATIONS_HELPER

### LINKED ISSUE #45919: [PHPUnitBridge] When only ignoreFile is passed in SYMFONY_DEPRECATIONS_HELPER, deprecations no longer fail tests
### Symfony version(s) affected

6.1.0

### Description

When only `ignoreFile` is passed in `SYMFONY_DEPRECATIONS_HELPER`, deprecations no longer fail tests, unless you also specifically set `max[total]=0`

### How to reproduce

Use an env var like (from phpunit.xml)

`<env name="SYMFONY_DEPRECATIONS_HELPER" value="ignoreFile=/var/www/d10/core/deprecation-ignore.txt"/>`

Run tests that have unignored deprecations reported;

The exit code of PHPUnit is 0, not 1 as you would expect.

Add `max[total]=0`

`<env name="SYMFONY_DEPRECATIONS_HELPER" value="max[total]=0&ignoreFile=/var/www/d10/core/deprecation-ignore.txt"/>`

Rerun tests;

The exit code of PHPUnit is  1.

### Possible Solution

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by mondrake**
| Q             | A
| ------------- | ---
| Branch?       | 6.1 
| Bug fix?      | yes
| New feature?  | no 
| Deprecations? | no 
| Tickets       | Fix #45919  
| License       | MIT

### CODE REVIEW

File: src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php [hunk 1]
```diff
@@ -96,7 +96,7 @@ private function __construct(array $thresholds = [], $regex = '', $verboseOutput
         }
         foreach ($groups as $group) {
             if (!isset($this->thresholds[$group])) {
-                $this->thresholds[$group] = 999999;
+                $this->thresholds[$group] = 0;
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php]: well, I'd rather add this after this foreach:
```php
$this->thresholds = $this->thresholds ?: ['total' => 0];
```

Comment (mondrake) [reply src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php]: If we have entered the constructor with an empty $thresholds array, at that point after the foreach, if I am not mistaken, `$this->thresholds` will be something like

```
[
  'total' => 999999,
  'indirect' => 999999, 
  'direct' => 999999, 
  'self' => 999999,
]
```

no?

Comment (nicolas-grekas) [reply src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php]: right! we should add this at the very beginning of the constructor then:
`$thresholds = $thresholds ?: ['total' => 0];`

Comment (mondrake) [reply src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php]: Done, thanks

**Comment by nicolas-grekas** [comment]
Thank you @mondrake.

**Pull request merged**